examples/example-storybook-v8-compareAfterShot/lostpixel.config.ts

```diff
@@ -0,0 +1,20 @@
+// @ts-nocheck
+
+import { CustomProjectConfig } from 'lost-pixel';
+
+export const config: CustomProjectConfig = {
+  storybookShots: {
+    storybookUrl: './storybook-static',
+    breakpoints: [320, 768],
+  },
+  generateOnly: true,
+  failOnDifference: true,
+
+  compareAfterShot: true,
+  flakynessRetries: 10,
+  // These times are greatly reduced due to compareAfterShot!
+  waitBetweenFlakynessRetries: 200,
+  waitBeforeScreenshot: 0,
+  waitForFirstRequest: 0,
+  waitForLastRequest: 0,
```

Comment:
**This is where this PR really shines!**

With the example-storybook-v8 with the default configs, it takes 10s to run -- and there are no retries.

If we use just the default flakynessRetries, it takes 20s as each page is screenshot twice.

With this PR, we can lower the delays a lot -- actually even more than the values here and with `shotConcurrency: 20` -- without being afraid of temporary failures as they aren't a big deal, as we can simply retry how many times we want.

**With compareAfterShot and this config, it takes 2s to run lostpixel**, without any retry actually made.

It allows us to be less conservative with the delays.